docs/_articles/en/einstein/einstein-inline.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Inline Automatic Completions
+lang: en
+---
+
+## Overview
+
+Use AI-based autocomplete to accept suggestions for repetitive, boilerplate code right in your editor.
+
+## Set Up Inline Autocomplete
+
+To use inline autocomplete, first activate the feature:
+
+1. Select **File** > **Preferences** > **Settings** (Windows or Linux) or **Code** > **Preferences** > **Settings** (macOS).
+2. Under **Einstein for Developers**, select **Enable Einstein Auto Completions**.
+
+## Receive your First Inline Completions
+
+Einstein for Developers autogenerates inline code completions suggestions. To start receiving inline completions:
+
+1. In Visual Studio Code, create a new Apex class (\*.cls) file, or open an existing one.
+
+2. In the Apex file, place your cursor inside a method. Einstein for Developers automatically suggests an entire method body in grayed text, as shown below. The exact suggestion may vary.
+
+![Inline Suggestions](./images/einstein-inline-create-account.png)
+
+To accept the entire suggestion, press Tab. If you have VS Code's **Inline Suggest** enabled, you can accept the next word of the suggestion using one of the following keyboard shortcuts:
+
+| Operating System | Accept Next Word |
+| ---------------- | ---------------- |
+| macOS            | ⌘→               |
+| Windows          | ⌥→               |
+| Linux            | ⌥→               |
+
+Note: If you don't see a suggestion, make sure Einstein Auto Completions are enabled.
+
+**_Note_**: Use the Option (⌥)\ hotkeys to manually generate autocomplete suggestions if you don't have the auto completions setting enabled.
```

Comment:
We're formatting both the `Notes` a bit differently, could we convert the bottom note to another table to share how to manually trigger the suggestions too? It could be a follow up to the previous `Note` as well....maybe something like `Even if you don't have the Auto Complete suggestion enabled, you can always trigger inline suggestions manually using the following hotkeys`